llvm/lib/Target/AIE/AIEHazardRecognizer.h

```diff
@@ -128,9 +136,11 @@ class AIEHazardRecognizer : public ScheduleHazardRecognizer {
   ///        use from the pre-RA scheduler, where detailed resource modelling
   ///        doesn't pay off.
   void emitInScoreboard(ResourceScoreboard<FuncUnitWrapper> &Scoreboard,
-                        const MCInstrDesc &Desc, int DeltaCycles) const;
+                        const MCInstrDesc &Desc, int DeltaCycles,
```

Comment:
Nit: I'd prefer `DeltaCycles` to remain the last attribute, I believe `Desc` and `MemoryBanks` make sense to be kept next to each other.